duckduckgo/BrowserServicesKit
# Update CodeQL Xcode version

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/1203301625297703/1205735703769411/f
iOS PR: Not required
macOS PR: Not required
What kind of version bump will this require?: None, this is a workflow file change

**Description**:

This PR updates the workflow files to use the same Xcode version across the board.



**Steps to test this PR**:
1. Make sure that PR checks succeed
2. Make sure that the CodeQL step has successfully run for this branch



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
